src/officehours_api/test_views.py

```diff
@@ -92,6 +97,29 @@ def test_cannot_change_backend_type_when_meeting_has_started(self):
             "Can't change backend_type once meeting is started!"
         )
 
+    def test_export_meeting_start_logs(self):
+        self.client.login(username='hosttwo', password='rohqtest')
+        # Start the meeting through the api to generate logs
+        response = self.client.post(f'/api/meetings/{self.meeting.id}/start')
+        self.assertEqual(response.status_code, status.HTTP_200_OK)
+        self.client.login(username='hostone', password='rohqtest')
+        # Start the meeting through the api to generate logs
+        response = self.client.get(f'/api/export_meeting_start_logs/')
+        self.assertEqual(response.status_code, status.HTTP_200_OK)
+        # This may change if the output of the export changes
+        self.assertEqual(len(response.content), 343)
```

Comment:
Tested with command `docker-compose run web python manage.py test officehours_api.test_views.MeetingTestCase`, and this test failed

======================================================================
FAIL: test_export_meeting_start_logs (officehours_api.test_views.MeetingTestCase)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/usr/src/app/officehours_api/test_views.py", line 110, in test_export_meeting_start_logs
    self.assertEqual(len(response.content), 343)
AssertionError: 289 != 343
 test case failed

Reply:
Yes, one of the unchecked TODO's I have is "Update failing tests or fix code". These are new tests I added and rather than verify against a string I figured checking the length was good. 

I didn't completely re-verify that this was working after making the SQL updates and expected it to fail. I just pushed the work I had so I could go over it with Justin and Jaydon. The new size looks like it could be correct but I need to verify the security checks were working.

Reply:
I wonder whether the assert can be based on the count of rows in the csv file, not on the length.

Reply:
Yeah that was a good idea, I was thinking the same. The data will vary in length and content because of some of the fields. I changed it to this.